src/components/Barrier.jsx

```diff
@@ -17,9 +17,13 @@ const Barrier = ({
     hidePriceLines,
     lineStyle,
     isInitialized,
-}) => (isInitialized &&
+}) => {
+    const chartContainer = document.getElementsByClassName('chartContainer')[0];
+    chartContainer.style.setProperty('--shade-color', shadeColor);
```

Comment:
this sets the shade color as part of the chartContainer. This property should be contained within the barrier itself. Preferably not using `getElementsByClassName` inside the render method as you are querying the DOM tree each time, and preferably setting the style property via react than manipulating the DOM directly.

Reply:
@brucebinary done!